package-lock.json

```diff
@@ -2547,11 +2547,13 @@
         },
```

Comment:
This is due to running on Windows - it shouldn't cause major issues.

Reply:
Interesting, I didn't get any of these changes.

Reply:
@nnekaO can you remove this, please?